schema-tool/README.md

```diff
@@ -0,0 +1,186 @@
+# Schema Tool for Scalar DB
+This tool creates Scalar DB schemas on Cosmos DB, DynamoDB, Cassandra and a JDBC database.
+  - For Cosmos DB, this tool creates databases(collections) and tables(containers), also inserts metadata which is required by Scalar DB.
+  - For DynamoDB, this tool creates tables named with the database names and table names, also inserts metadata which is required by Scalar DB.
+  - For Cassandra, this tool creates databases(keyspaces) and tables. You can specify the compaction strategy, the network topology strategy, and the replication factor as well.
+  - For a JDBC database, this tool creates databases(schemas, except Oracle) and tables, also inserts metadata which is required by Scalar DB.
+  - You don't have to add Scalar DB metadata for transactions. This tool automatically adds them when you set the `transaction` parameter `true` in your schema file.
+
+# Usage
+## Build & Run
+### Build application
+```console
+$ ./gradlew installDist
+```
+
+### Create tables
+Using config file based from Scalar DB. Sample config file can be found [here](sample_data/database.properties)
+```console
+$ ./build/install/schema-tool/bin/schema-tool --config <PATH_TO_CONFIG_FILE> -f schema.json
+```
+
+Using cli arguments for configuration
+```console
+# For Cosmos DB
+$ ./build/install/schema-tool/bin/schema-tool --cosmos -h <COSMOS_DB_ACCOUNT_URI> -p <COSMOS_DB_KEY> -f schema.json [-r BASE_RESOURCE_UNIT]
```

Comment:
Sorry, maybe the story wasn't clear enough but the options should be consistent for all the DBs. (as in the first requirement of the corresponding story)
So, for example,
if `scalar.db.username` matches  with `u`, then we must use `u` for all the databases to specify `scalar.db.username`. 

For database-specific parameters like  `-n`,  I would like you to design what alphabet to use properly.
I think it is ideal if we use only lower case characters.

Reply:
> we should stick to the same options for now, and change the options when new major version is released?

+1 (Sorry for the late response)